option cc_generic_services = true;


// 字段规则：required -> 字段只能也必须出现 1 次
// 字段规则：optional -> 字段可出现 0 次或1次
// 字段规则：repeated -> 字段可出现任意多次（包括 0）
// 类型：int32、int64、sint32、sint64、string、32-bit ....
// 字段编号：0 ~ 536870911（除去 19000 到 19999 之间的数字）
// 字段规则 类型 名称 = 字段编号;
message ProtocolBodyRequest
{
	optional string serviceName = 1;    //第一个参数
	optional uint32 methodIndex = 2;    //第二个参数
	optional uint32 callId = 3;         //...
	optional bytes content = 4;
}

message ProtocolBodyResponse
{
	optional uint32 callId = 1;
	optional bytes content = 2;
}